locales/en-US/app.ftl

```diff
@@ -47,7 +47,7 @@ Home--load-from-url-submit-button =
     .value = Load
 
 Home--documentation-button = Documentation
-Home--menu-button = Enable { -profiler-brand-name } Menu Button
+Home--menu-button = Enable Profiler Menu Button
```

Comment:
@julienw I actually had to revert this change I made in another PR. The brand name has `Firefox` in it so the texts are not same. Should I create another brand name for `Profiler`?

Reply:
I've seen that, I implemented a different solution, see https://github.com/firefox-devtools/profiler/pull/3204/commits/f66e8ff56fede7908f7d3d8ad1281ead7f881860#diff-e79a66b81b1c68868f86efe40a44847a1e6d395952fa4d2392e0aa4aede0c123R134-R139